application/src/constants/language/frCa.json

```diff
@@ -403,6 +403,7 @@
   "Syncing your information.": "Synchronisation de vos informations.",
   "Getting verification data.": "Obtenir des données de vérification.",
   "Getting identification data.": "Obtenir des données d'identification.",
+  "Getting extended transasction data.": "Obtenir des données d'identification.",
```

Comment:
This looks like the same translation as the line above. Do we have a process to go through to get translations, or can we just use something like Google Translate?

This applies to all translations files.